src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -759,12 +760,24 @@
      */
     public DataOverlayParameters dataOverlay = null;
 
+    /**
+     * Raptor can print all events when arriving at stops to stderrr. For developers only.
+     */
+    public Set<StopLocation> debugRaptorStops;
 
     /**
-     * Raptor can print all events when arriving at stops to system error. For developers only.
+     * Raptor can print all events for a path to stderr. For developers only.
+     *
+     * @see #debugRaptorPathFromStopIndex
      */
-    public DebugRaptor raptorDebuging = null;
+    public List<StopLocation> debugRaptorPath;
 
+    /**
+     * Stop index to print a Raptor paths to stderr. For developers only.
+     *
+     * @see #debugRaptorPath
+     */
+    public int debugRaptorPathFromStopIndex = 0;
```

Comment:
It would be nice if you can revert the changes above, feel free to improve the JavaDoc, but keep the:
```
public DebugRaptor raptorDebuging = null;
```